api/query/cache/service/search.go

```diff
@@ -114,6 +114,7 @@ func searchHandlerImpl(w http.ResponseWriter, r *http.Request) *searchError {
 				}
 			}
 			runPtr.ID = int64(id)
+			// nolint:errcheck // TODO: Fix errcheck lint error.
 			go idx.IngestRun(*runPtr)
```

Comment:
wrap this in an anonymous function and print a warning out like I mentioned in api/query/cache/backfill/backfill.go